jdbc/jooq/pom.xml

```diff
@@ -0,0 +1,70 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>spring-data-jdbc-jooq</artifactId>
+
+    <parent>
+        <groupId>org.springframework.data.examples</groupId>
+        <artifactId>spring-data-jdbc-examples</artifactId>
+        <version>2.0.0.BUILD-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <name>Spring Data JDBC - Usage with jOOQ</name>
+    <description>Sample project demonstrating Spring Data JDBC features</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.jooq</groupId>
+            <artifactId>jooq</artifactId>
+            <version>3.11.0</version>
```

Comment:
Avoid putting versions directly into dependencies. 

Ideally remove them completely. 
They might be already defined by an upstream `pom.xml`.

If they aren't extract them into a property.

Reply:
done